pkg/kubelet/pod_workers.go

```diff
@@ -28,7 +28,15 @@ import (
 	"github.com/golang/glog"
 )
 
-type syncPodFnType func(*api.Pod, *api.Pod, kubecontainer.Pod) error
+const (
+	minBackoffInterval = time.Second * 4
```

Comment:
Although any sleeps < 10s (the sync interval) won't have an effect since we don't get an update with less than that (if we're not behind, which is usually the case and most likely the case after a restart).